yt_dlp/extractor/webcamerapl.py

```diff
@@ -0,0 +1,42 @@
+import codecs
+
+from .common import InfoExtractor
+
+
+class WebcameraplIE(InfoExtractor):
+    _VALID_URL = r'https?://(?P<id>[\w-]+)\.webcamera\.pl'
+    _TESTS = [{
+        'url': 'https://warszawa-plac-zamkowy.webcamera.pl',
+        'info_dict': {
+            'id': 'warszawa-plac-zamkowy',
+            'ext': 'mp4',
+            'title': 'WIDOK NA PLAC ZAMKOWY W WARSZAWIE',
```

Comment:
```suggestion
            'title': r're:WIDOK NA PLAC ZAMKOWY W WARSZAWIE \d{4}-\d{2}-\d{2} \d{2}:\d{2}$',
```
yt-dlp adds timestamp to title for livestreams. Do same for the other test

Reply:
Sure, added.